eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Pool.hs

```diff
@@ -136,6 +136,7 @@ data PoolEvent era
 instance NFData (PoolEvent era)
 
 instance Era era => EncCBOR (ShelleyPoolPredFailure era) where
+  -- We can't use the Coders library here because we omit one of the fields during serialisation
```

Comment:
This statement isn't exactly true. We provide a specific failure message for `WrongCertificateTypePOOL` but we don't have to do that. 
That being said, we don't need to switch to Coder, but the comment isn't accurate, so that is the reason for my suggestion to remove the comment.

```suggestion
```